awx/main/models/ha.py

```diff
@@ -423,6 +423,13 @@ def on_instance_group_saved(sender, instance, created=False, raw=False, **kwargs
 
 @receiver(post_save, sender=Instance)
 def on_instance_saved(sender, instance, created=False, raw=False, **kwargs):
+    # TODO: this needs to be smarter. Did the port change, etc?
+    if settings.IS_K8S and created and (instance.node_type == 'execution' or instance.node_type == 'hop'):
```

Comment:
Optional picky: it would have been more compact to use `instance.node_type in (...)`